express/manifest.py

```diff
@@ -1,112 +1,113 @@
-"""
-Script for generating the dataset manifest that will be passed and updated through different
-components of the pipeline
-"""
+"""This module defines classes to represent an Express manifest."""
+
 import json
-from dataclasses import dataclass, field
-from enum import Enum
-from typing import List, Dict
+import pkgutil
+import typing as t
```

Comment:
Can we avoid one-letter imports in favor of `from typing import Dict, Union` etc

Reply:
I like `import typing as t` because:
- Importing every type directly clutters the namespace and obfuscate where an import is coming from (eg. `Iterable` can be imported from `typing` or from `collections`)
- Importing typing as a module and repeating `typing.` becomes long very quickly.

I'm not married to this approach though, and the important thing is probably to standardize on a way to do this across the repo.

My second choice would still be `import typing` instead of `from typing import ...`

Reply:
As discussed offline, let's go with `import typing as t`